client/src/components/embedding/index.tsx

```diff
@@ -28,6 +28,8 @@ type EmbeddingState = any;
   schema: (state as any).annoMatrix?.schema,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
   crossfilter: (state as any).obsCrossfilter,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
+  imageUnderlay: (state as any).imageUnderlay,
```

Comment:
state is explicitly any right now, you can can type the state arg as RootState and remove the casts